MbsAPI/fLmd.c

```diff
@@ -61,1186 +56,1266 @@ int clock_gettime(int clockid, struct timespec* tp)
+    pLmdControl->iOffsetEntries = newEntries;
 }
 //===============================================================
 void fLmdPrintBufferHeader(uint32_t iVerbose, sMbsBufferHeader* pMbsBufferHeader)
 {
-  if(iVerbose) {
-    if(pMbsBufferHeader) {
-      printf("BfHd: # %d, DataWords:%d Type:%08x Elements:%d sec:%d.%d MaxWords:%d\n",
-             pMbsBufferHeader->iBuffer,
-             pMbsBufferHeader->iUsedWords,
-             pMbsBufferHeader->iType,
-             pMbsBufferHeader->iElements,
-             pMbsBufferHeader->iTimeSpecSec,
-             pMbsBufferHeader->iTimeSpecNanoSec/1000,
-             pMbsBufferHeader->iMaxWords);
-    }
-  }
+    if (iVerbose) {
+        if (pMbsBufferHeader) {
+            printf("BfHd: # %d, DataWords:%d Type:%08x Elements:%d sec:%d.%d MaxWords:%d\n",
+                   pMbsBufferHeader->iBuffer,
+                   pMbsBufferHeader->iUsedWords,
+                   pMbsBufferHeader->iType,
+                   pMbsBufferHeader->iElements,
+                   pMbsBufferHeader->iTimeSpecSec,
+                   pMbsBufferHeader->iTimeSpecNanoSec / 1000,
+                   pMbsBufferHeader->iMaxWords);
+        }
+    }
 }
 //===============================================================
 void fLmdPrintFileHeader(uint32_t iVerbose, sMbsFileHeader* pMbsFileHeader)
 {
-  if(iVerbose) {
-    if(pMbsFileHeader) {
-      printf("FiHd: DataWords:%d Type:%d.%d Elements:%d sec:%d.%d MaxWords:%d Index: %llx[%d]\n",
-             pMbsFileHeader->iUsedWords,
-             pMbsFileHeader->iType&0xffff,
-             pMbsFileHeader->iType>>16,
-             pMbsFileHeader->iElements,
-             pMbsFileHeader->iTimeSpecSec,
-             pMbsFileHeader->iTimeSpecNanoSec/1000,
-             pMbsFileHeader->iMaxWords,
-             (long long unsigned int)(pMbsFileHeader->iTableOffset),
-             pMbsFileHeader->iOffsetSize);
-    }
-  }
+    if (iVerbose) {
+        if (pMbsFileHeader) {
+            printf("FiHd: DataWords:%d Type:%d.%d Elements:%d sec:%d.%d MaxWords:%d Index: %llx[%d]\n",
+                   pMbsFileHeader->iUsedWords,
+                   pMbsFileHeader->iType & 0xffff,
+                   pMbsFileHeader->iType >> 16,
+                   pMbsFileHeader->iElements,
+                   pMbsFileHeader->iTimeSpecSec,
+                   pMbsFileHeader->iTimeSpecNanoSec / 1000,
+                   pMbsFileHeader->iMaxWords,
+                   (long long unsigned int)(pMbsFileHeader->iTableOffset),
+                   pMbsFileHeader->iOffsetSize);
+        }
+    }
 }
 //===============================================================
 void fLmdPrintHeader(uint32_t iVerbose, sMbsHeader* pMbsHeader)
 {
-  if(iVerbose) {
-    if(pMbsHeader) {
-      printf("ElHd: words:%d type:%08x\n",
-             pMbsHeader->iWords,
-             pMbsHeader->iType);
+    if (iVerbose) {
+        if (pMbsHeader) {
+            printf("ElHd: words:%d type:%08x\n", pMbsHeader->iWords, pMbsHeader->iType);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [%d in format string (no. 1) requires 'int' but the argument type is 'unsigned int'.](https://app.codacy.com/manual/FairRootGroup/FairRoot/pullRequest?prid=5323596)